apache-php-mysql/0.4/httpd.conf

```diff
@@ -41,7 +41,7 @@ CustomLog "/var/log/httpd/access_log" common
         Require all granted
         AllowOverride All
 	    Options FollowSymlinks
-        DirectoryIndex index.php index.html hostingstart.html
+        DirectoryIndex error_localdb.php index.php index.html hostingstart.html
```

Comment:
If customer uploaded his apps to /home/site/wwwroot without removal of error_localdb.php, what would happen?

Reply:
to ensure app install without effect of error_localdb.php 
 - local mode: remove it with set DATABASE_USERNAME, DATABASE_PASSWORD
 - no local mode(now only remote): remove once it exists

Reply:
We shouldn't add a DirectoryIndex prior to index.php/index.html here.

Reply:
That would make user can't see his index.php/html

Reply:
if there is already a index.php(e.g. wordpress), app will begin to install directly even not enable local database